src/index.js

```diff
@@ -17,16 +18,31 @@ const getTabValue = path => {
     return null;
 };
 
+const rootElement = document.getElementById('root');
+
 const Main = ({ ytAPI }) => {
-    const [lastVideoId, setLastVideoId] = useState(null);
+    const [lastVideoId, setLastVideoId] = useState(undefined);
+    const [infoText, setInfoText] = useState(undefined);
+
+    const showInfo = (infoText, infoDuration, logToConsole = false) => {
```

Comment:
showInfo could be pass on with a context. See: https://reactjs.org/docs/context.html 

That way, you don't have to pass it along as prop everywhere. And you can call it use it anywhere.